examples/xgboost/create_poc_workpace.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+n_clients=$1
+
+if test -z "${n_clients}"
+then
+      echo "Usage: ./create_poc_workspace.sh [n_clients], e.g. ./create_poc_workspace.sh 8"
+      exit 1
+fi
+
+cur_dir=${PWD}
+workspace="xgboost_workspace_${n_clients}"
+
+# create POC startup kits
+cd "workspaces" || exit
+python3 -m nvflare.lighter.poc -n "${n_clients}" || exit
```

Comment:
we might want to change this to use the latest consolidate entry "nvflare", but that can be done in a separate PR.

Reply:
sure, this line is also used in other examples, we could do the change to all in a separate PR update